flow/peerdbenv/dynamicconf.go

```diff
@@ -506,6 +506,6 @@ func UpdatePeerDBMaintenanceModeEnabled(ctx context.Context, pool *pgxpool.Pool,
 	return UpdateDynamicSetting(ctx, pool, "PEERDB_MAINTENANCE_MODE_ENABLED", ptr.String(strconv.FormatBool(enabled)))
 }
 
-func PeerDBPKMEmptyBatchThrottleEnabled(ctx context.Context, env map[string]string) (bool, error) {
-	return dynamicConfBool(ctx, env, "PEERDB_PKM_EMPTY_BATCH_THROTTLE_ENABLED")
+func PeerDBPKMEmptyBatchThrottleThresholdSeconds(ctx context.Context, env map[string]string) (int64, error) {
```

Comment:
should probably get these duration settings to return `time.Duration`, but that'd be best as a followup PR